modules/rtpengine/rtpengine.c

```diff
@@ -228,6 +235,8 @@ static pid_t mypid;
 static unsigned int myseqn = 0;
 static str extra_id_pv_param = {NULL, 0};
 static char *setid_avp_param = NULL;
+static int hash_table_tout = 120;
```

Comment:
Is a default value of 2 minutes really all that useful? If a call lasts longer than 2 minutes, the final bye/delete won't be delivered to the correct RTP proxy any more, or do I misunderstand this value?